### PR TITLE
tiflash refactor: split use-tiflash into multiple docs

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
tiflash refactor: split use-tiflash into multiple docs

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/9452
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/use-tidb-to-read-tiflash.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
+title: 使用 TiDB 读取 TiFlash
+summary: 了解如何使用 TiDB 读取 TiFlash 副本。
+---
+
+# 使用 TiDB 读取 TiFlash
+
```
Comment (TomShawn) [inline tiflash/use-tidb-to-read-tiflash.md:7]:
```suggestion

本文档介绍如何使用 TiDB 读取 TiFlash 副本。

```

File: tiflash/tiflash-supported-pushdown-calculations.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* Limit：该算子对数据进行 limit 运算
+* Project：该算子对数据进行投影运算
+* HashJoin：该算子基于 [Hash Join](/explain-joins.md#hash-join) 算法对数据进行连接运算：
+    * 只有在 [MPP 模式](/tiflash/use-tiflash-mpp-mode.md)下才能被下推
+    * 支持的 Join 类型包括 Inner Join、Left Join、Semi Join、Anti Semi Join、Left Semi Join、Anti Left Semi Join
+    * 对于上述类型，既支持带等值条件的连接，也支持不带等值条件的连接（即 Cartesian Join）；在计算 Cartesian Join 时，只会使用 Broadcast 算法，而不会使用 Shuffle Hash Join 算法
+* Window：当前支持下推的窗口函数包括：row_number()、rank() 和 dense_rank()
+
+在 TiDB 中，算子之间会呈现树型组织结构。一个算子能下推到 TiFlash 的前提条件，是该算子的所有子算子都能下推到 TiFlash。因为大部分算子都包含有表达式计算，当且仅当一个算子所包含的所有表达式均支持下推到 TiFlash 时，该算子才有可能下推给 TiFlash。目前 TiFlash 支持下推的表达式包括：
+
```
Comment (qiancai) [inline tiflash/tiflash-supported-pushdown-calculations.md:28]: 以下是支持下推的表达式

File: tiflash/tiflash-supported-pushdown-calculations.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+* 字符串函数：`substr, char_length, replace, concat, concat_ws, left, right, ascii, length, trim, ltrim, rtrim, position, format, lower, ucase, upper, substring_index, lpad, rpad, strcmp, regexp`
+* 日期函数：`date_format, timestampdiff, from_unixtime, unix_timestamp(int), unix_timestamp(decimal), str_to_date(date), str_to_date(datetime), datediff, year, month, day, extract(datetime), date, hour, microsecond, minute, second, sysdate, date_add, date_sub, adddate, subdate, quarter, dayname, dayofmonth, dayofweek, dayofyear, last_day, monthname, to_seconds, to_days, from_days, weekofyear`
+* JSON 函数：`json_length`
+* 转换函数：`cast(int as double), cast(int as decimal), cast(int as string), cast(int as time), cast(double as int), cast(double as decimal), cast(double as string), cast(double as time), cast(string as int), cast(string as double), cast(string as decimal), cast(string as time), cast(decimal as int), cast(decimal as string), cast(decimal as time), cast(time as int), cast(time as decimal), cast(time as string), cast(time as real)`
+* 聚合函数：`min, max, sum, count, avg, approx_count_distinct, group_concat`
+* 其他函数：`inetntoa, inetaton, inet6ntoa, inet6aton`
+
+## 支持下推的表达式
```
Comment (qiancai) [inline tiflash/tiflash-supported-pushdown-calculations.md:39]:
```suggestion
## 下推限制
```

File: tiflash/tiflash-supported-pushdown-calculations.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+TiFlash 支持下推以下表达式：
+
```
Comment (qiancai) [inline tiflash/tiflash-supported-pushdown-calculations.md:42]:
```suggestion
```

**Comment by qiancai** [comment]
other LGTM

File: tiflash/tiflash-supported-pushdown-calculations.md (hunk 2 above)
Comment (TomShawn) [inline tiflash/tiflash-supported-pushdown-calculations.md:28]:
```suggestion
在 TiDB 中，算子之间会呈现树型组织结构。一个算子能下推到 TiFlash 的前提条件，是该算子的所有子算子都能下推到 TiFlash。因为大部分算子都包含有表达式计算，当且仅当一个算子所包含的所有表达式均支持下推到 TiFlash 时，该算子才有可能下推给 TiFlash。

## 支持下推的表达式

```

**Review by qiancai — APPROVED** [review]

**Pull request merged**